src/main/Cargo.toml

```diff
@@ -6,7 +6,7 @@ publish = false
 
 [lib]
 path = "lib.rs"
-crate-type = ["staticlib"]
+crate-type = ["rlib", "staticlib"]
```

Comment:
Probably worth a comment here that the `rlib` is there to enable doc-tests

Reply:
Added a comment.